bootstrap/resources/omp-apps.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: open-management-portal
+spec:
+  destination:
+    namespace: argo-cd 
+    server: https://kubernetes.default.svc
+  project: default
+  source:
+    path: "applications/"
+    repoURL: "https://github.com/rht-labs/open-management-portal-deployment.git"
+    targetRevision: "master"
+  syncPolicy:
+    automated:
+      prune: true
+      selfHeal: true
```

Comment:
Same as above regarding moving this to runtime config? Both `namespace` and `targetRevision` might have to be configured at runtime.